src/main/java/seedu/revision/logic/commands/main/CommandResult.java

```diff
@@ -38,6 +47,19 @@ public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, bool
         this.showHelp = showHelp;
         this.exit = exit;
         this.start = start;
+        this.showRestore = false;
+    }
+
+    /**
+     * Constructs a {@code CommandResult} with the specified fields.
+     */
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, boolean start, boolean restore, Model model) {
```

Comment:
Possible to use a less overloaded constructor of CommandResult to pass the model? Seems like got some information that the the command doesn't need like "Start". So your initialiser can be shorter than 
"CommandResult(SHOWING_CONFIRMATION_MESSAGE, false, false, false, true, model);" Since the first 3 will always be false

Can look into it ah I'm not sure why got so many overloaded instructions as well.